swagger.yaml

```diff
@@ -212,8 +212,8 @@ definitions:
                       description: Job staus
                       enum:
                         - Active
-                        - Leave without pay with benefits
-                        - Leave without pay without benefits
+                        - Leave without pay but with benefits
```

Comment:
This looks good, but don't forget to update these strings in the mapper as well.